### PR TITLE
Minor updates

### PR DESCRIPTION
**Pull request by katrin-freihofner**
we need to version++

### CODE REVIEW

File: docs/_pages/showcase.md [hunk 1]
```diff
@@ -64,15 +64,15 @@ hideCode: true
...
     <figure>
-      <img src="/assets/images/showcase/blog.ruxit.com.jpg" alt="blog.ruxit.com">
+      <img src="/assets/images/showcase/blog.dynatrace.com.jpg" alt="dynatrace.com/blog">
     </figure>
     <p>
       The fastest blog in the world uses Groundhog.
     </p>
     <p>
-      <a href="http://blog.ruxit.com" class="btn btn--primary">See it in action</a>
+      <a href="http://dynatrace.com/blog/" class="btn btn--primary">See it in action</a>
```
Comment (lara-aigmueller) [inline docs/_pages/showcase.md]: We could change this to `https`

File: src/button/samples/01-button-primary.html [hunk 2]
```diff
@@ -1 +1 @@
-<button role="button" type="button" class="btn btn--primary">Click me</button>
+<button role="button" type="button" class="btn btn--primary" href="#noop">Click me</button>
```
Comment (lara-aigmueller) [inline src/button/samples/01-button-primary.html]: There is no `href` attribute for the `<button>` element  :)

File: src/tile/samples/00-tile.html [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<button class="tile">
+<button href="#noop" class="tile">
```
Comment (lara-aigmueller) [inline src/tile/samples/00-tile.html]: See above: no `href` for `<button>`

File: docs/_pages/showcase.md (hunk 1 above)
Comment (ddprrt) [inline docs/_pages/showcase.md]: https://www.

**Comment by ddprrt** [comment]
What Lara said.

**Pull request merged**